xml/dhcp_vendor_class.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="dhcp_vendor_class" protocol="dhcp" database_type="service">
+  <!--
+    Fingerprint definitions that are matched against the string values in the
+    dhcp message vi_vendor_class field
+    The vi_vendor_class field can be found in client discover (1), request (3)
+    and inform (8) messages.
+  -->
+
+  <fingerprint pattern="^Mfg=(?:Fuji)?(?i:Xerox);Typ=(?:MFP|printer);Mod=(?:Xerox )?(\S+) ([a-zA-Z0-9]+)">
+    <description>Xerox Multifunction Printer</description>
+    <example hw.family="VersaLink" hw.model="C405">Mfg=Xerox;Typ=MFP;Mod=VersaLink C405;Ser=</example>
+    <example hw.family="AltaLink" hw.model="C8035">Mfg=Xerox;Typ=MFP;Mod=Xerox AltaLink C8035 Multifunction Printer;Ser=</example>
+    <example hw.family="WorkCentre" hw.model="3345">Mfg=XEROX;Typ=MFP;Mod=WorkCentre 3345;Ser=3BT844742;Loc=</example>
+    <example hw.family="Phaser" hw.model="6500DN">Mfg=FujiXerox;Typ=printer;Mod=Phaser 6500DN;Ser=YXE135427</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="Xerox"/>
+    <param pos="1" name="hw.family"/>
+    <param pos="2" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.family} {hw.model}"/>
+    <param pos="0" name="os.vendor" value="Xerox"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Mfg=DELL;Typ=Printer;Mod=Dell (\S+) Laser Printer;">
+    <description>DELL Laser Printer</description>
+    <example hw.model="B2360dn">Mfg=DELL;Typ=Printer;Mod=Dell B2360dn Laser Printer;Ser=</example>
```

Comment:
If you have an example with a serial number please go ahead and add the serial number value so that we can capture it in the future.  

My preference would be that all serial number values be modified so that they aren't a serial number that we've actually observed but ARE representative in content and length.